actions/check-whl-on-pypi/action.yml

```diff
@@ -22,7 +22,7 @@ runs:
         PACKAGE_VER=`echo "$WHL_NAME" | cut -d'-' -f2`
         EXISTING_VERS=`pip index --pre versions $PACKAGE_NAME`
         STATUS=0
-        if [ $(echo "$EXISTING_VERS" | grep "PACKAGE_VER," | wc -l) -gt 0 ]; then
+        if [ $(echo "$EXISTING_VERS" | grep "$PACKAGE_VER," | wc -l) -gt 0 ]; then
```

Comment:
don't you want the $ in the error msg string too?

Reply:
Nice catch! Updated, thanks!